cylc/flow/cfgspec/workflow.py

```diff
@@ -1490,6 +1490,33 @@ def upg(cfg, descr):
         ['scheduling', 'hold after cycle point']
     )
 
+    u.deprecate(
+        '8.0.0',
+        ['runtime', '__MANY__', 'suite state polling'],
+        ['runtime', '__MANY__', 'workflow state polling'],
+    )
+
+    for setting in [
+        'user',
+        'host',
+        'interval',
+        'max-polls',
+        'message',
+        'run-dir',
+        'verbose mode'
+    ]:
+        u.deprecate(
+            '8.0.0',
+            ['runtime', '__MANY__', 'suite state polling', setting],
+            ['runtime', '__MANY__', 'workflow state polling', setting],
+        )
```

Comment:
I have a feeling this is not needed, I thought
```python
    u.deprecate(
        '8.0.0',
        ['runtime', '__MANY__', 'suite state polling'],
        ['runtime', '__MANY__', 'workflow state polling'],
    )
```
above would take care of it.

Reply:
Yes you're right - thought I had tested that to check, but evidently not!